.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+
+## Ways to contribute
+
+There are many ways to contribute to Web Foundation, some of which are:
+
+- Filing [bug reports](https://github.com/Shopify/web-foundation/issues/new?template=BUG_REPORT.md)
+- Requesting new features or packages via [an issue](https://github.com/Shopify/web-foundation/issues/new/choose)
+  - Bringing up areas for enhancement
+- Hacking away on an issue from our [backlog](https://github.com/Shopify/web-foundation/issues)
+- Improving tests or documentation
+
+Want to contribute, but not sure how? Find us on Slack in `#web-foundation-tech`.
+
+## Development
+
+### Getting Started
+
+```bash
+dev clone web-foundation
+dev up
+```
+
+[what is dev?](#what-is-dev)
+
+### Getting productive
+
+We are adding documentation as we go in the [Web Foundation repo](https://github.com/Shopify/web-foundation). There you will find our [decision records](https://github.com/Shopify/web-foundation/tree/master/Decision%20records), [principles](https://github.com/Shopify/web-foundation/tree/master/Principles), [best practices](https://github.com/Shopify/web-foundation/tree/master/Best%20practices) and [styleguides](https://github.com/Shopify/web-foundation/tree/master/Styleguides) for writing and [testing](https://github.com/Shopify/web-foundation/blob/master/Best%20practices/Testing.md) different kinds of components.
+
+The [documentation](../documentation) directory in this repo covers the more granular technical aspects of this project. Of particular note for new folks are the following:
+
+- [Guides](../documentation/guides): a set of guides to help you get started developing with `web-foundation`. Of particular note for developers just starting on the project is our guide to [creating a new package](../documentation/guides/creating-a-new-package.md).
+- [FAQ](../documentation/FAQ.md): common questions about the project in general, as well as some of the technical pieces within.
+- [Resources](../documentation/resources.md): good resources for understanding this project’s tech stack.
+- [Getting started](../documentation/getting-started.md): some tools we recommend for getting the most out of this project.
+
+## Testing your changes in a local project
+
+To try out your changes in another locally cloned project, you can use `yarn tophat <package-name-without-@shopify-prefix> <relative-path-to-project>`. Using this command rather than `yarn link` will set up a watcher let you make changes without needing to rerun any commands.
+
+Example: To test my changes to `@shopify/react-form-state` in my local project named `cool-proj`, I would run `yarn tophat react-form-state ../path/to/cool-proj`.
+
+More usage instructions on the `tophat` command can be [found here](https://github.com/Shopify/webgen/blob/master/docs/TOPHAT.md).
+
+### Emoji commits
```

Comment:
We've had a few discussions about this in the past. I think the conclusion was to not enforce it. Maybe remove?